snocker/README.md

```diff
@@ -0,0 +1,41 @@
+## Containerized snorkel
+This fork of the snorkel repo has been modified slightly to be build and deployed in a Docker container. No clone of this repository is necessary, that will be performed in a the Docker image build. Just download the app in the [*snocker* directory](https://github.com/HazyResearch/snorkel/tree/master/snocker) or create a local dockerfile and copy the contents into that file.
+
+Make sure Docker is installed on your run environment and that the Dockerfile is available on the run envinment in a directory named app. Then build the image.
+
+```
+$ docker build -f app/dockerfile -t snocker:0.6.2 .
+```
+
+Now you can run the app. Link the home directory to somewhere easy to find.
+
+```
+$ docker run -it --name snocker -p 8887:8887 -v ~/some/local/dir/mapped/to/snokel/projects:/home/snorkel/projects snocker:0.6.2
+## hit esc key sequence: ctrl+p+q
+```
+
+You can now execute a bash command in the running container, move the snorkel directory, and run snorkel.
+
+```
+$ docker exec -it snocker bash
+root@ab1234:/# cd /home/snorkel
+root@ab1234:/snorkel# ./run.sh
```

Comment:
It seems that `./run.sh` should be set as ENTRYPOINT in Dockerfile.

Reply:
I have no problem with this change. It does assume that a user will know what the heck is going on ones the run dialog is asking to install coreNLP and returns a token. without explicitly initiating the process, one wont have a chance to explore the directory structure, or make the decision to launch.